packages/appsync-emulator-serverless/lambdaSource.js

```diff
@@ -23,9 +23,9 @@ const lambdaSource = async (
   // Default to empty string, path.join will resolve this automatically
   let buildPrefix = '';
 
-  // Check if the modulePrefix configuration is set
-  if (custom['appsync-emulator'] && custom['appsync-emulator'].buildPrefix) {
-    ({ buildPrefix } = custom['appsync-emulator']);
+  // Check if the buildPrefix configuration is set
+  if (custom['appsync-offline'] && custom['appsync-offline'].buildPrefix) {
```

Comment:
Why do you want to change this?

Reply:
@amalow -- do you have any thoughts here?

Reply:
This would be my fault, it appears the README uses `appsync-offline` instead of `appsync-emulator` hence the confusion.

As this project is unrelated to `appsync-offline` I'd suggest keeping it as `appsync-emulator` and updating the README to reflect this.

Reply:
@RuBAN-GT ☝️